components/main_page/wikis/arenafps/main_page_layout_data.lua

```diff
@@ -0,0 +1,203 @@
+---
+-- @Liquipedia
+-- wiki=arenafps
+-- page=Module:MainPageLayout/data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local CONTENT = {
+	usefulArticles = {
+		heading = 'Useful Articles',
+		body = '{{Liquipedia:Useful Articles}}',
+		padding = true,
+		boxid = 1503,
+	},
+	wantToHelp = {
+		heading = 'Want To Help?',
+		body = '{{Liquipedia:Want_to_help}}',
+		padding = true,
+		boxid = 1504,
+	},
+	transfers = {
+		heading = 'Transfers',
+		body = '{{Transfer List|limit=15}}\n<div style{{=}}"display:block; text-align:center; padding:0.5em;">\n' ..
+			'<div style{{=}}"display:inline; float:left; font-style:italic;">\'\'[[#Top|Back to top]]\'\'</div>\n' ..
+			'<div style{{=}}"display:inline; float:right;" class="plainlinks smalledit">' ..
+			'&#91;[{{FULLURL:Player Transfers/{{CURRENTYEAR}}/1st_Quarter|action=edit}} edit]&#93;</div>\n' ..
+			'<div style{{=}}"white-space:nowrap; display:inline; margin:0 10px font-size:15px; font-style:italic;">' ..
+			'[[Portal:Transfers|See more transfers]]<span style="font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[Transfer query]]<span style{{=}}"font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[lpcommons:Special:RunQuery/Transfer|Input Form]]' ..
+			'<span style="font-style:normal; padding:0 5px;">&#8226;</span>' ..
+			'[[Portal:Rumours|Rumours]]</center></div>\n</div>',
+		boxid = 1509,
+	},
+	thisDay = {
+		heading = 'This day in Arena FPS <small id="this-day-date" style = "margin-left: 5px">' ..
+			'({{#time:F}} {{Ordinal|{{#time:j}}}})</small>',
+		body = '{{Liquipedia:This day}}',
+		padding = true,
+		boxid = 1510,
+	},
+	specialEvents = {
+		noPanel = true,
+		body = '{{Liquipedia:Special Event}}',
+	},
+	filterButtons = {
+		noPanel = true,
+		body = '<div style{{=}}"width:100%;margin-bottom:8px;">' ..
+			'{{#invoke:Lua|invoke|module=FilterButtons|fn=getFromConfig}}</div>',
+	},
+	matches = {
+		heading = 'Matches',
+		body = '{{#invoke:Lua|invoke|module=Widget/Factory|fn=fromTemplate|widget=Match/Ticker/Container}}' ..
+			'<div style{{=}}"white-space:nowrap; display: block; margin:0 10px; ' ..
+			'font-size:15px; font-style:italic; text-align:center;">[[Liquipedia:Matches|See more matches]]</div>',
+		padding = true,
+		boxid = 1507,
+	},
+	tournaments = {
+		heading = 'Tournaments',
+		body = '{{#invoke:Lua|invoke|module=Widget/Factory|fn=fromTemplate|widget=Tournaments/Ticker' ..
+			'|upcomingDays=40|completedDays=40}}',
+		padding = true,
+		boxid = 1508,
+	},
+}
+
+return {
+	banner = {
+		lightmode = 'Arenafps-lightmode.svg',
+		darkmode = 'Arenafps-darkmode.svg',
+	},
+	metadesc = 'The Arena FPS esports wiki covering everything from players, teams and transfers, ' ..
+		'to tournaments and results, maps, and weapons.',
```

Comment:
is there maps and weapons on arenafps wiki?

Reply:
There are map and weapon pages yeah eg:

https://liquipedia.net/arenafps/Corrupted_Keep/Quake_Champions
https://liquipedia.net/arenafps/Diabotical/PnCR